src/Admin.php

```diff
@@ -26,6 +26,13 @@ class Admin {
    */
   const CRON_EVENT_REVISION_CLEANUP_RETAIN_DAYS = 90;
 
+  /**
+   * Cron event name for delete orphan post meta.
+   *
+   * @var string
+   */
+  const CRON_EVENT_DELETE_ORPHAN_POST_META = Plugin::PREFIX . '/cron/delete-orphan-post-meta';
```

Comment:
Let's make this consistent with the existing constant name, i.e. `CRON_EVENT_POSTMETA_CLEANUP`

We can rename the value accordingly.